requirements_test.txt

```diff
@@ -1,34 +0,0 @@
-# Note: using < here to reduce breakages
-# These should be automatically kept up to date by dependabot
```

Comment:
Not sure if this is still the case with `pyproject.toml`? maybe `dependabot` will just work without any config change?